src/lib/openjp2/openjpeg.h

```diff
@@ -129,6 +129,8 @@ typedef uint64_t OPJ_UINT64;
 
 typedef int64_t  OPJ_OFF_T; /* 64-bit file offset type */
 
+typedef unsigned int OPJ_BITFIELD;
```

Comment:
This type is only used by internal headers. Please do not define it in openjpeg public header.